src/frontend/http.rs

```diff
@@ -596,9 +621,19 @@ pub fn filters(
         .then(upload)
         .map(into_response);
 
+    // Health-check/readiness probe
+    let ctx = context;
+    let health_route = warp::path!("readyz")
```

Comment:
Still not sure whether this is readyz, healthz or both.

Reply:
IIUC healthz (liveness) is for when something internal has gone wrong in the system and we need it to be restarted. E.g. we check that some background worker thread is still running and if it's crashed, we say we're unhealthy.

I think we can make `healthz` and `readyz` do the same thing for now.